### DISCUSSION TITLE
Should focus to native input wrapped by Controller first before focus to native input not.

### DISCUSSION THREAD

**Discussion by sonphung205** [react-hook-form/react-hook-form#6223] [RFC]
I am using native input with Controller to keep validate with overwriten onchange after first "onSubmit". But after first submit, it's focus to the native input not wrapped by Controller first. It's should focus by order.

Refer to this sandbox pls https://codesandbox.io/s/beautiful-wood-ouy8x?file=/src/App.js

Steps 
1:  Scroll to bottom
2:  Hit complete.
3:  Cursor should focus to the first input, but it's focus to second  input

Sorry about my terrible engilsh! First time using github Q&A and Codesandbox, forgive me!

**Comment by bluebill1049** [comment]
native input will register ahead of Controller, if you need the correct focus order, wrap all with Controller so the order remains the same.

**Reply by sonphung205** [reply]
Does it's all the same? I mean sometimes i need to work with list of checkbox  wrapped by Controller and it's so messy.

**Reply by SarKurd** [reply]
Hey @bluebill1049 is this still the case in V7? any alternative workaround?
